conf/rocks-block.ini.template

```diff
@@ -0,0 +1,386 @@
+# This is a RocksDB option file.
```

Comment:
Do we have to define the full set of keys in this file? Is it possible to just define the necessary and leave the rest?

Reply:
These are created using the current settings (from the java definitions) in Alluxio. My thinking was that users could use the current settings as a start point, then if they wanted to change a single thing at a time they could.

I could also do something like start RocksDB with no configuration settings, then compare the difference to the files generated by java, then only include the changes that the java generated files created, with the assumption that the rest of the options would just take the defaults when not defined. This would create shorter config files for example if preferred.

Reply:
I agree if we can shorten this list to a list of non-default values, that would be great.

Reply:
I removed many of them, I left mostly the ones that I have been testing or plan to test.